docs/orchestration/dcos/README.md

```diff
@@ -35,5 +35,7 @@ $ dcos package uninstall minio
 ```
 
 ### Explore Further
+- [Advanced installation of Minio on DC/OS GUI](https://github.com/dcos/examples/blob/master/minio/1.9/README.md#minio-installation-using-gui)
 - [Minio Erasure Code QuickStart Guide](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide)
```

Comment:
We can probably move this up, under the section `Minio installation on DC/OS GUI` . Thoughts?